app/lib/vudu.js

```diff
@@ -20,152 +20,177 @@ const colors = {
 };
 
 const c = v.config({ colors });
-//LAYOUT
+// LAYOUT
+
+const numbers = [0, 50, 100];
+
+numbers.forEach((n) => {
+  c[`maxHeight${n}`] = { maxHeight: `${n}px` };
+  c[`minHeight${n}`] = { minHeight: `${n}px` };
+  c[`maxWidth${n}`] = { maxWidth: `${n}px` };
+  c[`minWidth${n}`] = { minWidth: `${n}px` };
+  c[`t${n}`] = { top: `${n}px` };
+  c[`r${n}`] = { right: `${n}px` };
+  c[`b${n}`] = { bottom: `${n}px` };
+  c[`l${n}`] = { left: `${n}px` };
+});
+
 c.mdFlex = {
-  display: 'initial'
-}
+  display: 'initial',
+};
 
 c.mdFlex[breakpoints.md] = {
-  display: 'flex'
-}
+  display: 'flex',
+};
+
+c.relative = {
```

Comment:
I think I left a console.log in the application route, so it should appear in the console already